python/cuml/common/base.pyx

```diff
@@ -108,15 +108,13 @@ class Base:
         run different models concurrently in different streams by creating
         handles in several streams.
         If it is None, a new one is created just for this class.
-    verbosity : int
+    verbose : int or boolean (default = False)
         Sets logging level. It must be one of `cuml.common.logger.LEVEL_*`.
```

Comment:
since the log level variables have been lowercased, better to update this documentation accordingly as well?